osu.Game/Overlays/BeatmapSetOverlay.cs

```diff
@@ -44,20 +45,25 @@ public BeatmapSetOverlay()
             Info info;
             CommentsSection comments;
 
-            Child = new FillFlowContainer
+            Child = new PopoverContainer
```

Comment:
Why is this not included in `CommentsContainer` instead?

Reply:
When there is only one comment, there is not enough space for the popover:
![изображение](https://user-images.githubusercontent.com/53872073/196994993-6111a295-6ed3-4a2e-bcf6-d32b563233d2.png)
I was not sure that it will work correctly in such cases, so made the whole overlay the container.